packages/cli/src/commands/deviceprofiles/translations/delete.ts

```diff
@@ -0,0 +1,68 @@
+import {DeviceProfile, LocaleReference} from '@smartthings/core-sdk'
+
+import { NestedSelectingAPICommand2 } from '@smartthings/cli-lib'
+
+
+export default class DeviceProfileTranslationDeleteCommand extends NestedSelectingAPICommand2<DeviceProfile, LocaleReference> {
```

Comment:
`DeviceProfileTranslation` (using singular) is inconsistent from other command names added in this change.

Reply:
Fixed